src/Components/Onboarding/Steps/Genes/SuggestedGenes.tsx

```diff
@@ -1,10 +1,70 @@
 import * as React from "react"
-import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { commitMutation, createFragmentContainer, graphql, QueryRenderer, RelayProp } from "react-relay"
+import { RecordSourceSelectorProxy, SelectorData } from "relay-runtime"
 
 import { ContextConsumer, ContextProps } from "../../../Artsy"
 import ItemLink from "../../ItemLink"
 
-class SuggestedGenesContent extends React.Component<RelayProps, null> {
+export interface RelayProps {
+  suggested_genes: [
+    {
+      id: string | null
+      _id: string | null
```

Comment:
I _think_ you probably meant `__id` (dunderscore) here?

Reply:
We actually intended to use `_id` in preparation for the `excluded_gene_ids` argument. It only accepts the unique `id` and not the `slug`. Normally I wouldn't add anything like this, but given that we will start the story as soon as this PR is merged, I think that's a good reminder for us to use mongoid `id`s for `excluded_gene_ids`.

Reply:
Gotcha 👍